catalogue/webapp/components/WorkSearchNoResults/WorkSearchNoResults.tsx

```diff
@@ -0,0 +1,46 @@
+import Space from '@weco/common/views/components/styled/Space';
+import { font, grid, classNames } from '@weco/common/utils/classnames';
+import styled from 'styled-components';
+import { FunctionComponent } from 'react';
+
+type Props = {
+  query: string;
+  hasFilters: boolean;
+};
+
+const QuerySpan = styled.span.attrs({
+  className: classNames({
+    [font('hnm', 2)]: true,
+  }),
+})`
+  fontweight: 400;
+`;
+
+const WorkSearchNoResults: FunctionComponent<Props> = ({
```

Comment:
This is going to be used for `/works` and `/images` so a name that covers both would probably be better – `CollectionsSearchNoResults` maybe?

Reply:
good idea

Reply:
@davidpmccormick  how about SearchNoResults ?